README.md

```diff
@@ -76,29 +76,16 @@ restart as well.
 
 You can pull our Docker images from [Docker Hub](https://hub.docker.com/r/timescale/timescaledb/).
 
-```bash
-docker pull timescale/timescaledb:latest
-```
+To run, simply issue the following command:
 
-To run, you'll need to specify a directory where data should be
-stored/mounted from on the host machine. For example, if you want
-to store the data in `/your/data/dir` on the host machine:
 ```bash
-docker run -d \
-  --name timescaledb \
-  -v /your/data/dir:/var/lib/postgresql/data \
-  -p 5432:5432 \
```

Comment:
Well, if you made that decision, then fine. But, FWIW, I think we should use Docker functionality (i.e., volumes) as intended and configured in the image. If someone is serious about data safety they would understand the semantics of Docker before running production workloads. Further, like directories, Docker volumes need to be explicitly deleted as  `"A Docker data volume persists after a container is deleted"`. Simplifying this run command and making it similar to the instructions for the main Postgres image might also alleviate some of the confusion that people have been experiencing.